pom.xml

```diff
@@ -29,13 +30,14 @@
 
     <properties>
         <!-- project version -->
-        <revision>2.0.3-SNAPSHOT</revision>
+        <revision>2.1.0-SNAPSHOT</revision>
```

Comment:
Only this line should change.

Reply:
the file was not formatted properly. If you want I can create a separate MR, but the changes are in format only, so I actually count that as trivial and don't think this should be done in another MR.

Reply:
as discussed lets make two commits.